app/units/units.css

```diff
@@ -119,6 +123,16 @@ span.icon {
 
 #filter {
     margin-right: 10px;
+    background-color: transparent;
+    border-radius: 50%; 
+    padding: 10px;
+    cursor: pointer; 
+    user-select: none;
+    transition: background-color 0.3s ease; 
+}
+
+#filter:hover {
+    background-color: rgba(0, 0, 0, 0.2); 
```

Comment:
As mentioned in my previous comment, "please use 0.1 opacity for the shadow" :) Everything else looks great - thanks!

Reply:
Ok I will make the changes

Reply:
I have changed the opacity and commit the changes into minor-uiEnhancements branch 
@jennifermtan  You can approve now. let me know if any changes needed.
<img width="960" alt="Screenshot 2024-06-03 at 8 14 40 PM" src="https://github.com/GoodGameRuler/unitrack_v3/assets/58948980/76e57e3f-e4c6-4402-882b-f7719d799ed3">